lib/actions/ui.js

```diff
@@ -208,16 +209,30 @@ export function moveTo(i) {
     });
   };
 }
+function getEditCommand(shell, isWin) {
+  if (isWin && shell.includes('bash')) {
+    return [
+      ' nano .hyper.js',
+      'echo Attempting to open .hyper.js with nano'
+    ];
+  } else if (isWin) {
+    return [
+      ' start notepad "%userprofile%\\.hyper.js"',
+      'echo Attempting to open .hyper.js with notepad'
+    ];
+  }
+  return [
+    // eslint-disable-next-line no-template-curly-in-string
+    ' bash -c \'exec env ${EDITOR:=nano} ~/.hyper.js\'',
+    'echo Attempting to open ~/.hyper.js with your \\$EDITOR'
+  ];
+}
 
 export function showPreferences() {
   const isWin = process.platform === 'win32';
-  // eslint-disable-next-line no-template-curly-in-string
-  const command = isWin ? ' start notepad "%userprofile%\\.hyper.js"' : ' bash -c \'exec env ${EDITOR:=nano} ~/.hyper.js\'';
-  const message = [];
-  message.push(isWin ?
-    ' echo Attempting to open ^%userprofile^%\\.hyper.js with notepad' :
-    ' echo Attempting to open ~/.hyper.js with your \\$EDITOR');
-  message.push(' echo If it fails, open it manually with your favorite editor!');
+  const plugins = remote.require('./plugins');
+  const {shell} = plugins.getDecoratedConfig();
+  const message = getEditCommand(shell, isWin);
```

Comment:
since you pass isWin use ` process.platform === 'win32'` instead of defining it.